tests/app/filter.test.js

```diff
@@ -31,3 +31,20 @@ test("scope filter updates markers", async ({ page }) => {
   await checkScope("Main Street/Special");
   await checkScope(["Transit Oriented", "City Center/Business District"]);
 });
+
+test("population filter updates markers", async ({ page }) => {
+  await loadMap(page);
+  const left = await page.$(".left-slider");
+  const right = await page.$(".right-slider");
+
+  const testSlider = async (slider, moveTo) => {
+    const before = await getNumCities(page);
+    await slider.fill(moveTo);
+    const after = await getNumCities(page);
+    expect(before > after).toBe(true);
+  };
+
+  await testSlider(left, "5");
```

Comment:
What does this number 5 correspond to? Like what is the unit? You could address this by giving a better name to `moveTo` so it's "self-documenting" code, like `moveToValue` or `movePixelsAmount`.